File: rest/test-credentials/test-sms-messages-example-1/test-sms-messages-example-1.4.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "your_auth_token";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    
    var sms = twilio.SendSmsMessage("+15005550006", "+14108675309", "All in the game, yo", "");
    
    Console.WriteLine(sms.Sid);
  }
}

File: rest/test-credentials/test-sms-messages-example-1/test-sms-messages-example-1.4.x.php
<?php

// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/twilio-php/Services/Twilio.php'; // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$authToken = "your_auth_token";
$client = new Services_Twilio($accountSid, $authToken);

$message = $client->account->messages->create(
    array(
        'To'   => '+15005550006',
        'From' => '+14108675310',
        'Body' => 'All in the game, yo'
    )
);

echo $message->sid;
